dev/github/run_tests.sh

```diff
@@ -38,7 +48,7 @@ PATH_BACKUP=${PATH}
 JAVA_HOME=/usr/local/openjdk-8
 PATH=$JAVA_HOME/bin:$PATH
 mvn -Duser.home=/home/jenkins -T 4C clean install -Dfindbugs.skip -Dcheckstyle.skip -DskipTests -Dmaven.javadoc.skip \
--Dlicense.skip -Dsort.skip ${mvn_args}
+-Dlicense.skip -Dsort.skip ${mvn_args} ${mvn_compile_args}
```

Comment:
```suggestion
-Dlicense.skip -Dsort.skip ${mvn_compile_args}
```
do not use the old one